apps/example/lib/main.dart

```diff
@@ -20,17 +20,52 @@ class App extends StatelessWidget {
   const App({super.key});
 
   @override
-  Widget build(BuildContext context) => MaterialApp(
-        builder: DevicePreview.appBuilder,
-        locale: DevicePreview.locale(context),
-        supportedLocales: const [
-          Locale("en", "US"),
-          Locale("nl", "NL"),
-        ],
-        theme: ThemeData.light(),
-        darkTheme: ThemeData.dark(),
-        home: LocationsUserStory(
-          options: LocationsOptions(),
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      builder: DevicePreview.appBuilder,
+      locale: DevicePreview.locale(context),
+      supportedLocales: const [
+        Locale("en", "US"),
+        Locale("nl", "NL"),
+      ],
+      theme: ThemeData.light(),
+      darkTheme: ThemeData.dark(),
+      home: LocationsUserStory(
+        options: LocationsOptions(
+          respositoryInterface: LocationsLocalRepository(density: 7),
+          mapOptions: LocationsMapOptions(
+            zoom: 7,
+            initialLocation: const Location(
+              latitude: 52.2056435,
+              longitude: 5.2,
+            ),
+            controlsOptions: LocationsMapControlsOptions(controls: [
+              MapControl(
+                icon: Icons.pin_drop,
+                onPressed: (controller) => controller.rotate(0),
+              ),
+              MapControl(
+                icon: Icons.gps_fixed,
+                onPressed: (_) {},
+              ),
+              MapControl(
```

Comment:
These should not be defined by the consuming app but rather be pre-defined. For example setting MapControl.zoomControls() would add zoomControls. You could allow custom controls later but that should not be required for base functionality